docker/DockerBuild.sh

```diff
@@ -146,18 +149,31 @@ BUILD_ARGS+=("--build-arg" "UID=${UID_ARG}"
             "--build-arg" "GID=${GID_ARG}"
             "--build-arg" "MINIFI_VERSION=${MINIFI_VERSION}"
             "--build-arg" "DUMP_LOCATION=${DUMP_LOCATION}"
-            "--build-arg" "DISTRO_NAME=${DISTRO_NAME}")
+            "--build-arg" "DISTRO_NAME=${DISTRO_NAME}"
+            "--build-arg" "DOCKER_SKIP_TESTS=${DOCKER_SKIP_TESTS}")
 
-if [ -n "${DOCKER_CCACHE_DUMP_LOCATION}" ]; then
-  DOCKER_BUILDKIT=1 docker build "${BUILD_ARGS[@]}" -f ${DOCKERFILE} --target build -t minifi_build ..
+if [ -n "${DISTRO_NAME}" ]; then
```

Comment:
why can't we (or why don't we want to) do the distro-based builds in the same way as the plain docker build?

Reply:
In the plain docker build we have multiple stages and use the `--target build` to create the build image then copy the ccache files from that image if needed, then continue with the second stage for the released image. In the other distro builds we don't have separate build and release images, because we don't use those images for running MiNiFi only get the built binaries from it.

Reply:
Could we add stages to all Dockerfiles so we can use them in the same way?

Reply:
I would rather not, stages are usually used if we need one or more helper images to create the final image, this way the layers of the helper stages only created as temporary layers that can be cleaned up afterwards. In the centos and other distro docker images we do not really have any logically distinguished stages and I would remove these stages in the focal, fedora, bionic Dockerfiles as well in the future.